packages/untp-playground/src/app/page.tsx

```diff
@@ -20,13 +24,25 @@ export default function Home() {
   const [testResults, setTestResults] = useState<{
     [key in PermittedCredentialType]?: TestStep[];
   }>({});
+  const [fileCount, setFileCount] = useState(0);
+  const { dispatchError, errors, setIsDetailsOpen } = useError();
+
+  const shouldDisplayUploadDetailBtn =
+    errors &&
+    errors.length > 0 &&
+    ((fileCount === 1 && Object.keys(testResults).length === 0) ||
+      (fileCount > 1 && Object.keys(testResults).length !== 0) ||
+      (fileCount > 1 && Object.keys(testResults).length === 0));
```

Comment:
Aren't these two lines the same as: `fileCount > 1` ? (I mean, either the length is not equal to 0 or it's equal to zero)